Sources/Danger/GitHubDSL.swift

```diff
@@ -189,7 +189,7 @@ extension GitHub {
     /// A GitHub user account.
     public struct User: Decodable, Equatable {
         enum CodingKeys: String, CodingKey {
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 192;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)